### PR TITLE
Fixed qiskit_integration notebook (gate rebase)

### PR DESCRIPTION
**Pull request by lmondada**
The original error in this notebook is fixed by https://github.com/CQCL/pytket-extensions/pull/196. There was another error with incompatible gatesets which I have fixed by using `default_compilation_pass` instead of `FullPeepholeOptimise`.

### CODE REVIEW

File: examples/python/qiskit_integration.py [hunk 1]
```diff
@@ -68,23 +68,14 @@ def vqe_solve(op, maxiter, quantum_instance):
 #
 # We can now run the VQE algorithm. In this example we use only 50 iterations, but greater accuracy may be achieved by increasing this number:
 
+vqe_solve(H2_op, 50, qi)
+
 print("VQE result:", vqe_solve(H2_op, 50, qi))
```
Comment (cqc-alec) [inline examples/python/qiskit_integration.py:73]: Seems we are calling `vqe_solve()` twice,

File: examples/python/qiskit_integration.py [hunk 2]
```diff
@@ -68,23 +68,14 @@ def vqe_solve(op, maxiter, quantum_instance):
...
-from qiskit.transpiler.passes import Unroller
-
-qis_backend2 = TketBackend(b_emu, FullPeepholeOptimise())
-qi2 = QuantumInstance(
-    qis_backend2,
-    pass_manager=PassManager(Unroller(["cx", "h", "rx", "ry", "rz"])),
-    shots=8192,
-    wait=0.1,
-)
+# Another way to improve the accuracy of results is to apply optimisations to the circuit in an attempt to reduce the overall noise. When we construct our qiskit backend, we can pass in a pytket compilation pass as an additional parameter. There is a wide range of options here; we recommend the device-specific default compilation pass, provided by each tket backend. This pass will ensure that all the hardware constrains of the device are met. We can enable tket's most agressive optimisation level by setting the parameter `optimisation_level=2`.
```
Comment (cqc-alec) [inline examples/python/qiskit_integration.py:75]:
```suggestion
# Another way to improve the accuracy of results is to apply optimisations to the circuit in an attempt to reduce the overall noise. When we construct our qiskit backend, we can pass in a pytket compilation pass as an additional parameter. There is a wide range of options here; we recommend the device-specific default compilation pass, provided by each tket backend. This pass will ensure that all the hardware constraints of the device are met. We can enable tket's most aggressive optimisation level by setting the parameter `optimisation_level=2`.
```

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: examples/python/qiskit_integration.py (hunk 1 above)
Comment (lmondada) [reply examples/python/qiskit_integration.py]: Fixed, thanks.

File: examples/python/qiskit_integration.py (hunk 2 above)
Comment (lmondada) [reply examples/python/qiskit_integration.py:73]: Urgh, a day of debugging has made me very tired... Thanks!

**Review by cqc-alec — APPROVED** [review]
Thanks!

**Pull request merged**